### PR TITLE
Code style: Use canonical naming and formatting

### PR DESCRIPTION
**Pull request by felixrabe**
This change is [Reviewable](https://reviewable.io/reviews/servo/servo/21282)

### CODE REVIEW

File: ports/servo/browser.rs [hunk 1]
```diff
@@ -273,9 +273,9 @@ impl Browser {
                         self.event_queue.push(WindowEvent::SendError(browser_id, reason));
                     }
                 }
-                EmbedderMsg::AllowNavigation(_url, response_chan) => {
-                    if let Err(e) = response_chan.send(true) {
-                        warn!("Failed to send allow_navigation() response: {}", e);
+                EmbedderMsg::AllowNavigation(_url, sender) => {
+                    if let Err(e) = sender.send(true) {
+                        warn!("Failed to send AllowNavigation response: {}", e);
                     };
```
Comment (emilio) [inline ports/servo/browser.rs]: nit: While at it you may as well remove this stray semicolon :)

**Review by emilio — APPROVED** [review]
Thanks!

**Comment by jdm** [comment]
@bors-servo retry
* busted CI

**Comment by jdm** [comment]
@bors-servo retry
* #21067

**Comment by jdm** [comment]
@bors-servo retry
* #21295

**Comment by Manishearth** [comment]
going to temporarily close this while I test some things

**Comment by felixrabe** [comment]
lol I start to get the feeling that this tiny nitpick of mine is about to bring up some PTSD regarding CI infrastructure 🤭

**Comment by Manishearth** [comment]
@bors-servo retry

 - virtualenv disappeared ?

**Comment by jdm** [comment]
@bors-servo retry
* cleaned out the build machine

**Pull request merged**